src/locales/fr/PAGES/SUMMARY.json

```
@@ -0,0 +1,19 @@
+{
+    "TITLE": "Étape 5: Résultat du rapport de l’évaluation",
+    "INTRO": "Fournir des informations supplémentaires sur l’évaluation que vous souhaitez voir figurer dans le rapport. Les résultats de l’évaluation que vous avez saisis à l’étape précédente sont inscrits ci-dessous dans la section « Résultats détaillés de l’audit » pour examen et pour vous aider à rédiger le résumé.",
+    "RESOURCE_LINK_NAME": "Étape 5: Résultat du rapport de l’évaluation",
+    "LABEL_TITLE": "Titre du rapport",
+    "TITLE_PREFIX": "Rapport pour",
+    "INF_TITLE": "Donnez un titre au rapport d’évaluation de l’accessibilité du site Web. Par exemple : <br>« Rapport d’évaluation pour une organisation donnée »<br> « Analyse de l’accessibilité de la boutique en ligne et suggestions d’amélioration ».",
+    "LABEL_COMMISSIONER": "Commandé par",
+    "INF_COMMISSIONER": "La personne, l’équipe, l’organisation, le service interne ou toute autre entité qui a commandé l’évaluation.",
+    "LABEL_CREATOR": "Évaluateur",
+    "INF_CREATOR": "La personne, l’équipe, l’organisation, le service interne ou toute autre entité responsable de la réalisation de l’évaluation.",
+    "LABEL_DATE": "Date de l’évaluation",
+    "INF_DATE": "Indiquer la date d’achèvement ou la période de cette évaluation. Vous pouvez utiliser n’importe quel format de date. Écrire le mois en toutes lettres pour que ce soit clair au niveau international. (Le format de date 01-02-2022 est ambigu - il s’agit du 2 janvier dans certains endroits et du 1er février dans d’autres).",
+    "LABEL_SUMMARY": "Synthèse",
+    "INF_SUMMARY": "Fournir un bref résumé des constats de l’évaluation pour donner un aperçu des résultats. Par exemple, décrivez l’accessibilité globale du site Web et les observations clés que vous avez faites lors de l’évaluation, tels que les problèmes qui se répètent fréquemment.",
+    "LABEL_SPECIFICS": "Registre des spécificités de l’évaluation (facultatif)",
+    "INF_SPECIFICS": "Le WCAG-EM suggère d’archiver les pages Web auditées. Pour plus d’informations, voir <a href={{link_em_5b}}>WCAG-EM Étape 5.b: Enregistrer les spécificités de l’évaluation</a>. Vous pouvez utiliser ce champ de texte pour enregistrer les outils d’évaluation, les navigateurs Web, les technologies d’assistance, les autres logiciels et les méthodes utilisés pour l’évaluation. Ce que vous saisissez ici sera inclus dans le rapport généré. Après avoir téléchargé le rapport, vous pouvez supprimer ou modifier ce champ dans le fichier HTML avant de le soumettre.",
```

```suggestion
    "INF_SPECIFICS": "WCAG-EM suggère que vous archiviez les pages Web auditées. Pour plus d’informations, voir <a href={{link_em_5b}}>WCAG-EM Étape 5.b&nbsp;: Enregistrer les détails de l’évaluation</a>. Vous pouvez utiliser ce champ de texte pour enregistrer les outils d’évaluation, les navigateurs Web, les technologies d’assistance, les autres logiciels et les méthodes utilisés pour l’évaluation. Ce que vous saisissez ici sera inclus dans le rapport généré. Après avoir téléchargé le rapport, vous pouvez supprimer ou modifier ces informations dans le fichier HTML avant de soumettre le rapport.",
```

Based on @ericporlier's PR.  Isn't "détails de l'évaluation" a better candidate than "spécificités de l'évaluation"? This part is less about "what was peculiar in this evaluation" and more about "the exact details of this evaluation: the web pages that were audited, the tools and methodology used, etc".